plugins/modules/remote_management/redfish/ilo_redfish_info.py

```diff
@@ -0,0 +1,190 @@
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: ilo_redfish_info
+short_description: Gathers server information through iLO using Redfish APIs
+description:
+  - Builds Redfish URIs locally and sends them to iLO to
+    get information back.
+  - For use with HPE iLO operations that require Redfish OEM extensions.
+options:
+  category:
+    required: true
+    description:
+      - List of categories to execute on iLO.
+    type: list
+    elements: str
+  command:
+    required: true
+    description:
+      - List of commands to execute on iLO.
+    type: list
+    elements: str
+  baseuri:
+    required: true
+    description:
+      - Base URI of iLO.
+    type: str
+  username:
+    description:
+      - User for authentication with iLO.
+    type: str
+  password:
+    description:
+      - Password for authentication with iLO.
+    type: str
+  auth_token:
+    description:
+      - Security token for authentication with iLO.
+    type: str
+    version_added: 2.3.0
+  timeout:
+    description:
+      - Timeout in seconds for URL requests to iLO.
+    default: 10
+    type: int
+
+requirements:
+    - "python >= 3.8"
+    - "ansible >= 3.2"
+author:
+    - "Bhavya B (@bhavya06)"
+'''
+
+EXAMPLES = '''
+  - name: Get iLO Sessions
+    community.general.ilo_redfish_info:
+      category: Sessions
+      command: GetiLOSessions
+      baseuri: "{{ baseuri }}"
+      username: "{{ username }}"
+      password: "{{ password }}"
+    register: result_sessions
+'''
+RETURN = '''
+ilo_redfish_info:
+    description: Returns iLO sessions.
+    type: dict
+    contains:
+        GetiLOSessions:
+            description: Returns the iLO session msg and whether the function executed successfully.
+            type: dict
+            contains:
+                ret:
+                    description: Check variable to see if the information was succesfully retrived.
+                    type: str
+                msg:
+                    description: Information of all active iLO sessions.
+                    type: dict
+                    elements: dict
+                    contains:
+                        Description:
+                            description: Provides a description of the resource.
+                            type: str
+                        Id:
+                            description: The sessionId.
+                            type: str
+                        Name:
+                            description: The name of the resource.
+                            type: str
+                        UserName:
+                            description: Name to use to log in to the management processor.
+                            type: str
+    returned: always
+    sample: Dictionary of iLO Sessions
```

Comment:
```suggestion
```
Either include an actual sample, or nothing. This shouldn't be another description.